configs/templates/skylab-dev/spack.yaml

```diff
@@ -27,7 +27,7 @@ spack:
     # Individual packages
     - bacio@2.4.1
     - bison@3.8.2
-    - bufr@11.7.0
+    - bufr@11.7.1
```

Comment:
As opposed to hpc-stack-dev (the outlier), skylab-dev needs this update, thanks.

Reply:
Left this in